styles/main.css

```diff
@@ -356,20 +357,23 @@ input[type="checkbox"]{
         position: absolute;
         top: 15px;
     }
+
+    .burger,
+    .burger::after,
+    .burger::before {
+        transition: all 200ms ease ;
+    }
+

```

Comment:
Set the transition on the un-transitioned elements so that the animation applies when it goes from its default state to any other state